include/aos/common/tools/time.hpp

```diff
@@ -253,6 +259,36 @@ class Time {
         return log;
     }
 
+    /**
+     * Converts time into a string.
+     * Result string has the following format: yyyyMMddhhmmss
+     *
+     * @result RetWithError<StaticString<cTimeStrLength>>.
+     */
+    RetWithError<StaticString<cTimeStrLength>> ToString() const
```

Comment:
Better to put time conversion into separate commit.

Reply:
done